Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_fd.nut

```diff
@@ -968,6 +968,9 @@ bool function runWave( int waveIndex, bool shouldDoBuyTime )
 		// is this in the right place? do we want to be adding for each player?
 		// this function is called "Set" but in reality it is "Add"
 		SetJoinInProgressBonus( GetCurrentPlaylistVarInt( "fd_money_per_round" ,600 ) )
+		if(!IsValidPlayer(player)) {
+			return false
```

Comment:
It's much better to use `continue` instead of `return false`.
Because we still need to do the samething in other player.

Reply:
I see, understood